### DISCUSSION TITLE
How I can disable running on issues

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [actions/stale#1123] [General]
I need this to run only on PRs, how I can disable issues?

**Comment by Piedone** [comment]
I'd have the same question. What I found for now is only that I've set `days-before-issue-stale: -1` but this only prevents issues being marked stale. Still, issues will be checked via the API.

**Comment by justinmchase** [comment]
I came here to ask the exact same thing, I only want this to apply to PR's and completely ignore issues.

**Comment by KornevNikita** [comment]
It'd be good to have the same, but for PRs - I'd like to check only issues. Still no solution?
